worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -231,6 +235,7 @@ private Path createOutputDirectory(Path outputDirPath, Job job) {
     private void deleteExistingDirectory(Path outputDirPath, Job job) {
         final File[] files = outputDirPath.toFile().listFiles(getFilenameFilter());
 
+        assert files != null;
```

Comment:
Wouldn't it be better to throw a runtime exception? Quartz catches those and logs them in a standard way.